### PR TITLE
[ACS-6565] ADW - Document List View Header - UX Bugs

### PR DESCRIPTION
**Pull request by jacekpluta**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/ACS-6565

**What is the new behaviour?**
Fixes according to the issue in JIRA.
image

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.scss [hunk 1]
```diff
@@ -65,3 +65,7 @@
     min-width: 260px;
     padding: 0;
 }
+
+.margin-left-auto {
```
Comment (MichalKinas) [inline lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.scss:69]: ADF related classes should be prefixed with `adf`

**Review by MichalKinas — COMMENTED** [review]
The rest LGTM :D please also get in touch with someone from HxP to make sure we're not introducing breaking changes on their side, as datatable is a shared component

File: lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.scss (hunk 1 above)
Comment (jacekpluta) [reply lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.scss]: fixed

**Review by MichalKinas — APPROVED** [review]

**Review by eromano — APPROVED** [review]
LGTM @popovicsandras can you give it a look as well?

**Review by popovicsandras — APPROVED** [review]

File: lib/core/src/lib/datatable/components/datatable/datatable.component.html [hunk 2]
```diff
@@ -125,14 +126,15 @@
             >
                 <ng-container *ngIf="mainActionTemplate">
                     <button
+                        class="adf-datatable__actions-cell-button"
```
Comment (AleksanderSklorz) [inline lib/core/src/lib/datatable/components/datatable/datatable.component.html:129]: is that class needed? I can't see styles for it

File: lib/core/src/lib/datatable/components/datatable/datatable.component.ts [hunk 3]
```diff
@@ -1009,6 +1009,10 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
         (sorting.key === col.key || sorting.key === col.sortingKey) &&
         sorting.direction?.toLocaleLowerCase() === direction;
     };
+
+    get isResizing() {
```
Comment (AleksanderSklorz) [inline lib/core/src/lib/datatable/components/datatable/datatable.component.ts:1013]: could you please add return type?

**Review by DenysVuika — APPROVED** [review]
looks good, but please review the comments

**Review by AleksanderSklorz — APPROVED** [review]

File: lib/core/src/lib/datatable/components/datatable/datatable.component.ts (hunk 3 above)
Comment (jacekpluta) [reply lib/core/src/lib/datatable/components/datatable/datatable.component.ts]: done

File: lib/core/src/lib/datatable/components/datatable/datatable.component.html (hunk 2 above)
Comment (jacekpluta) [reply lib/core/src/lib/datatable/components/datatable/datatable.component.html]: good catch, removed

**Pull request merged**